frontend/src/model/releases/ReleaseForm.tsx

```diff
@@ -41,6 +50,9 @@ type ReleaseFormProps = {
   filesMetadata: FileWithMetadata[]
   onFilesMetadataChange: (value: FileWithMetadata[]) => void
   onImageListChange: (value: FlattenedModelImage[]) => void
+  currentFileUploadProgress?: FileUploadProgress
+  uploadedFiles?: string[]
```

Comment:
To simplify the conditional logic, can we make this required? We can provide an empty array in place of `undefined`.